frontend/src/AddTask.js

```diff
@@ -19,12 +20,13 @@ export const AddTask = () => {
 
   return (
     <form
-      onSubmit={handleSubmit(({ title }) => {
+      onSubmit={handleSubmit(({ title, effort }) => {
         insertTask.mutate(
-          { title },
+          { title, effort },
           {
             onSuccess: () => {
               resetField("title");
+              resetField("effort");
             },
```

Comment:
I believe it should be good to have a callback for error handling as well

Reply:
Good point 👍